cds_dojson/marc21/fields/books/multipart.py

```diff
@@ -173,8 +285,77 @@ def number_of_volumes(self, key, value):
     raise IgnoreKey('number_of_volumes')
 
 
-@model.over('book_series', '^490__')
+@model.over('book_series', '^490__', override=True)
+@filter_list_values
 @for_each_value
 def book_series(self, key, value):
-    """Match barcodes to volumes."""
-    base_book_series(self, key, value)
+    """Translates issns stored in the record."""
+    val_a = clean_val('a', value, str)
+    val_x = clean_val('x', value, str)
+
+    if val_a:
+        if 'title' in self:
+            raise UnexpectedValue(subfield='a', message=' series has already a title')
+        if val_x:
+            raise UnexpectedValue(subfield='x', message=' should not be present')
+        self['title'] = val_a
+    else:
+        raise UnexpectedValue(subfield='x')
+
+    raise IgnoreKey('book_series')
+
+
+@model.over('is_singleton', '^596__')
+def is_singleton(self, key, value):
+    """Multivolume kind."""
+    val_a = clean_val('a', value, str)
+    _migration = self['_migration']
+    if val_a == 'MULTIVOLUMES-1':
+        parsed = True
+    elif val_a == 'MULTIVOLUMES-X' or val_a == 'MULTIVOLUMES-x':
+        parsed = False
+    elif val_a == 'MULTIVOLUMES-MANUAL':
+        raise Exception('This record should not be migrated!')
+    else:
+        raise UnexpectedValue(
+            subfield='a', message=' unrecognized migration multipart tag'
+        )
+    _migration['is_singleton'] = parsed
+    raise IgnoreKey('is_singleton')
+
+
+@model.over('serial_id', '^597__')
```

Comment:
`multipart_id`, serial is another type of series (there are two only multipart monograph and serial)